build.sh

```diff
@@ -0,0 +1,94 @@
+#!/bin/bash
+multipleBuildsPath="../builds/"
+individualBuildAllFiles="build/*"
+
+craBatteryConsiderateLoading="cra-battery-considerate-loading"
+craDeviceClassAwareLoading="cra-device-class-aware-loading"
+craMemoryConsiderateLoading="cra-memory-considerate-loading"
+craMemoryConsiderateLoadingSketchfab="cra-memory-considerate-loading-sketchfab"
+craNetworkAwareCodeSplitting="cra-network-aware-code-splitting"
+craNetworkAwareComponent="cra-network-aware-component"
+craNetworkAwareDataFetching="cra-network-aware-data-fetching"
+craUaAwareCodeSplitting="cra-ua-aware-code-splitting"
+reactMovieNetworkAwareComponents="react-movie-network-aware-components"
+reactShrineNetworkAwareCodeSplitting="react-shrine-network-aware-code-splitting"
+
+cd $craBatteryConsiderateLoading
+rm -rf node_modules build
+npm install
+npm run build
+mkdir -p "${multipleBuildsPath}${craBatteryConsiderateLoading}"
+cp -r $individualBuildAllFiles "${multipleBuildsPath}${craBatteryConsiderateLoading}"
+cd ..
+
+cd $craDeviceClassAwareLoading
```

Comment:
Am I correct in saying if a build for a particular sub-project fails, we will bail out of building the others?

Reply:
I'm afraid I have not tried if we can skip the failure and keep going with the rest of builds.
Let me try by deliberately making an error to one build.

Reply:
Based on what I've tested, we can skip the failure and keep going with the rest of builds.